examples/test_cheque.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+
```

Comment:
can you add the comment before the actual code in the examples:
```python
# begin-doc-include
```

That will make the code show up in the generated documentation.